examples/server/server.cpp

```diff
@@ -1745,7 +1745,10 @@ struct server_context {
 
                     // Erase token cache
                     const size_t n_erased = slot->cache_tokens.size();
-                    llama_kv_cache_seq_rm(ctx, slot->id + 1, -1, -1);
+                    if (!llama_kv_cache_seq_rm(ctx, slot->id + 1, -1, -1)) {
```

Comment:
Noted. Will this remain true in the future?

Reply:
Yes, removing whole sequences should never fail.
In the future, it might return something else than a `bool`, though (like the resulting last `pos`, to more gracefully handle recurrent state checkpoints), but it's only ever going to be useful when partially deleting sequences.

Reply:
Ok, adjusted, and added note to `llama_kv_cache_seq_rm`.